db_revisions/versions/ccc50ec18a7e_add_submission_upload_malformed_to_state_enum.py

```diff
@@ -0,0 +1,46 @@
+"""add SUBMISSION_UPLOAD_MALFORMED to state enum
+
+Revision ID: ccc50ec18a7e
+Revises: 11dfc8200daa
+Create Date: 2024-04-07 01:02:13.310717
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op, context
+
+
+# revision identifiers, used by Alembic.
+revision: str = "ccc50ec18a7e"
+down_revision: Union[str, None] = "fb46d55283d6"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+old_options = (
+    "SUBMISSION_UPLOADED",
+    "VALIDATION_IN_PROGRESS",
+    "VALIDATION_WITH_ERRORS",
+    "VALIDATION_WITH_WARNINGS",
+    "VALIDATION_SUCCESSFUL",
+    "SUBMISSION_ACCEPTED",
+    "SUBMISSION_STARTED",
+)
+new_options = sorted(old_options + ("SUBMISSION_UPLOAD_MALFORMED",))
+
```

Comment:
This will fail in Postgres because it doesn't like the double quotes.  See 7a1b7eab0167_accept_updates.py but basically you have to change these to single quotes, and before the lists turn formatting checks off so black doesn't revert it to double quotes, then after the lists turn formatting back on.

Reply:
Oh one last thing, you have to convert the sorted list to a tuple again.  The statement `op.execute(f"CREATE TYPE submissionstate AS ENUM{new_options}")` is actually relying on the formatted string of the tuple to build the correct CREATE sql statement, and using just sorted will cause it to build a sql statement with [ ] instead of ( )